react/test/components/ReviewForm.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReviewForm from '../../src/components/ReviewForm';
+
+describe('ReviewForm', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    jasmineEnzyme();
```

Comment:
You could clean this up by moving this beforeEach jasmineEnzyme() to your testHelper file